### PR TITLE
Update note on 3.0 release timeline

### PR DESCRIPTION
**Pull request by srowen**

### CODE REVIEW

**Comment by dongjoon-hyun** [comment]
Thank you for making this update, @srowen !

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM for everything (New release windows and written LTS version policy).

**Comment by dongjoon-hyun** [comment]
Since this is not a hard deadline, I'll merge this for now.
We can adjust time-to-time if needed.

File: site/versioning-policy.html [hunk 1]
```diff
@@ -251,30 +251,30 @@ <h2>Release Cadence</h2>
...
     <tr>
-      <td>Late Aug 2018</td>
-      <td>&#160;</td>
+      <td>Late Dec 2019</td>
       <td>QA period. Focus on bug fixes, tests, stability and docs. Generally, no new features merged.</td>
     </tr>
     <tr>
-      <td>Early Sep 2018</td>
-      <td>&#160;</td>
+      <td>Jan 2019</td>
```
Comment (dongjoon-hyun) [inline site/versioning-policy.html:277]: Oops. Sorry, I missed this. This should be `2020`.

Comment (srowen) [reply site/versioning-policy.html:277]: Ack, oops. I'm sure people will get the idea but feel free to hotfix it or whatever

Comment (dongjoon-hyun) [reply site/versioning-policy.html:277]: Sure. I'll make a PR for this.

**Pull request merged**